src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV2.cs

```diff
@@ -177,15 +190,15 @@ public class DownloadPipelineArtifactTaskV2_0_0 : PipelineArtifactTaskPluginBase
                 {
                     if (pipelineVersionToDownload == pipelineVersionToDownloadLatest)
                     {
-                        pipelineId = await this.GetPipelineIdAsync(context, pipelineDefinition, pipelineVersionToDownload, projectName, tagsInput);
```

Comment:
We can consider passing the resultFilter to the function and set it default to Succeed so that we don't need to always pass two bool variable here.

Reply:
Done.